app-service-environment-serviceplan-logicapp/azuredeploy.json

```diff
@@ -8,6 +8,12 @@
         "description": "The name to use for the new AppService Environment"
       }
     },
+    "location": {
```

Comment:
When doing so deployment fails. That's why I changed it back to a parameter and commit it. Maybe a fix is needed in the parsing of the template in order to support this.